RMSD.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+import os, sys, glob
+import openeye.oechem as oechem
+import openmoltools
+
+#-------------------- Functions---------------------#
+
+
+def RMSD(ffRef, ffList, molSet, molName):
+    """
+    From a set of reference molecules and query molecules, the RMSD values are
+     computed and wrote into a file.  
+   
+    Parameters
+    ---------
+    ffRef: str - the reference forcefield
+    ffList: str - the query forcefield
+    molSet: str - set of molecule
+    molName: str - name of the molecules
+   
+    Returns
+    -------
+    RMSD values for each molecules with the defined ff
```

Comment:
This function doesn't return anything. 

returning is specific, it means I could call the function and have that value, for example if this actually returned the rmsd you could call
```
rmsd = RMSD(...)
print(rmsd)
# prints a number like 0.001 or something
```
This prints to the command line, but doesn't return anything